helpers/auth.js

```diff
@@ -0,0 +1,17 @@
+const passport = require("passport");
+
+const auth = (req, res, next) => {
+  passport.authenticate("jwt", { session: false }, (err, user) => {
+    if (!user || err) {
+      return res.status(401).json({
+        status: "Unauthorized",
+        code: 401,
+        message: "Not authorized",
+      });
+    }
+    req.user = user;
```

Comment:
Jezeli dobrze rozumiem, to na tym poziomie `req.user` to `user`, ktory jest obiektem literalnym o ksztalcie `{ id, email }`. Jesli tam jest `id`, to dlaczego w wielu miejscach uzywasz `email`, zeby wydobyc `id` usera? 🤔 Np. w `ctrlUsers.js` masz sporo zabaw tego typu:
```javascript
const { email } = req.body;
  try {
    const user = await service.getUser(email);
    await User.findByIdAndUpdate(user.id, { token: null });
```

A moze mozna po prostu: `await User.findByIdAndUpdate(req.user.id, { token: null });` ? 🤔 

Daj znac,bo moze sie myle :)